tests/intel_fpga/vec_sum.py

```diff
@@ -43,9 +40,33 @@ def sum(i: _[0:N]):
 
     Z_exp = X + Y + Z
 
-    #compute expected result
-    vec_sum(X, Y, Z)
+    sdfg = vec_sum.to_sdfg()
+
+    if args.vectorize_first == "true":
+        transformations = [
+            dace.transformation.dataflow.vectorization.Vectorization,
+            dace.transformation.interstate.fpga_transform_sdfg.FPGATransformSDFG
+        ]
+        transformation_options = [{}, {
+            "propagate_parent": True,
+            "postamble": False
+        }]
+    elif args.vectorize_first == "false":
+        transformations = [
+            dace.transformation.interstate.fpga_transform_sdfg.
+            FPGATransformSDFG,
+            dace.transformation.dataflow.vectorization.Vectorization
+        ]
+        transformation_options = [{}, {
+            "propagate_parent": True,
+            "postamble": False
+        }]
+
+    for t, to in zip(transformations, transformation_options):
+        sdfg.apply_transformations(t, to)
```

Comment:
As far as I know, the API would work if you just fed in the two lists.

Reply:
Yeah I think you are right, it wasn't working for me initially, but for a different reason. Will change